### PR TITLE
DOC: Add link to NumPy PDF docs

### LINKED ISSUE #19303: DOC: There is no easy way to find the NumPy PDF docs.
They are located at `https://numpy.org/doc/`, but there is no visible link to that page.

### PR DESCRIPTION
**Pull request by yashasvimisra2798**
Fixes #19303

Added the line ``For a PDF version of the documentation, go to https://numpy.org/doc/.``  under NumPy v1.21 Manual heading
Please check

### CODE REVIEW

File: doc/source/_templates/indexcontent.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
 {% block body %}
 <h1>{{ docstitle|e }}</h1>
 <p>
-  Welcome! This is the documentation for NumPy {{ release|e }}
-  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}.
+  Welcome! This is the documentation for NumPy {{ release|e }} 
+  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. For a PDF version of the documentation, go to https://numpy.org/doc/.
```
Comment (melissawm) [inline doc/source/_templates/indexcontent.html:10]: To make this a clickable link, you can do
```suggestion
  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. For a PDF version of the documentation, go to <a href="https://numpy.org/doc/">https://numpy.org/doc/</a>.
```

Comment (yashasvimisra2798) [reply doc/source/_templates/indexcontent.html]: Thanks, @melissawm, I have made the changes please check.

File: doc/source/_templates/indexcontent.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
 {% block body %}
 <h1>{{ docstitle|e }}</h1>
 <p>
-  Welcome! This is the documentation for NumPy {{ release|e }}
-  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}.
+  Welcome! This is the documentation for NumPy {{ release|e }} 
+  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. For a PDF version of the documentation, go to <a href="https://numpy.org/doc/">https://numpy.org/doc/</a>.
```
Comment (mattip) [inline doc/source/_templates/indexcontent.html:10]:
```suggestion
  {%- if last_updated %}, last updated {{ last_updated|e }}{% endif %}. 
  For a PDF version of the documentation, go to <a href="https://numpy.org/doc/">https://numpy.org/doc/</a>.
```

**Comment by mattip** [comment]
[here](https://20971-908607-gh.circle-artifacts.com/0/doc/build/html/index.html) is the rendered page. LGTM, with a small formatting nit about line length.

**Comment by mattip** [comment]
Thanks @yashasvimisra2798

**Pull request merged**